back/spec/lib/participation_method/native_survey_spec.rb

```diff
@@ -20,6 +20,70 @@
     end
   end
 
+  describe '#create_default_form!' do
+    let(:multiloc_service) { instance_double MultilocService }
+
+    before do
+      allow(MultilocService).to receive(:new).and_return multiloc_service
+      allow(multiloc_service).to receive(
+        :i18n_to_multiloc
+      ).with(
+        'form_builder.default_select_field.title'
+      ).and_return({
+        'en' => 'Default question',
+        'fr-FR' => 'Question par défaut',
+        'nl-NL' => 'Standaardvraag'
+      })
+      allow(multiloc_service).to receive(
+        :i18n_to_multiloc
+      ).with(
+        'form_builder.default_select_field.option1'
+      ).and_return({
+        'en' => 'First option',
+        'fr-FR' => 'Première option',
+        'nl-NL' => 'Eerste optie'
+      })
+      allow(multiloc_service).to receive(
+        :i18n_to_multiloc
+      ).with(
+        'form_builder.default_select_field.option2'
+      ).and_return({
+        'en' => 'Second option',
+        'fr-FR' => 'Deuxième option',
+        'nl-NL' => 'Tweede optie'
+      })
+    end
+
+    it 'persists a default form for the participation context' do
+      expect(participation_context.custom_form).to be_nil
+
+      participation_method.create_default_form!
+
+      participation_context.reload
```

Comment:
Do you think it could make sense to do a `reload` inside the `.create_default_form!` method? Since this is a method of the instance, I would expect the changes to this instance to be applied.

Reply:
Done.